### PR TITLE
add save tiff pipeline

### PR DESCRIPTION
**Pull request by CJ-Wright**
Attn: @sbillinge

### CODE REVIEW

File: xpdan/pipelines/save_tiff.py [hunk 1]
```diff
@@ -1,185 +1,157 @@
...
+fg_dark_query.filter(lambda x: x == []).sink(lambda x: print('No dark found!'))
+(FromEventStream('event', ('data', image_name),
+                 fg_dark_query.filter(lambda x: x != [])
+                 .map(lambda x: x if not isinstance(x, list) else x[0])
+                 .map(lambda x: x.documents(fill=True)).flatten()
+                 ).map(np.float32)
+ .connect(raw_foreground_dark))
+
+# Get foreground
+FromEventStream('event', ('seq_num',), source, stream_name='seq_num'
```
Comment (sbillinge) [inline xpdan/pipelines/save_tiff.py]: remove....no masking so don't need seq_num

File: xpdan/pipelines/save_tiff.py [hunk 2]
```diff
@@ -1,185 +1,157 @@
...
+                ).connect(img_counter)
+(FromEventStream('event', ('data', image_name), source, principle=True,
+                 stream_name='raw_foreground').map(np.float32)
+ .connect(raw_foreground))
+
+# Save out calibration data to special place
+h_timestamp = start_timestamp.map(_timestampstr)
+
+raw_source.starsink(StartStopCallback())
+# raw_source.visualize(os.path.expanduser('~/mystream.png'), source_node=True)
```
Comment (sbillinge) [inline xpdan/pipelines/save_tiff.py]: clean out here?

**Comment by CJ-Wright** [comment]
The errors are due to orch.

**Comment by CJ-Wright** [comment]
This already went in, but if you find any issues please let me know so I can patch it!

**Pull request merged**